### PR TITLE
Log in file

### PR DESCRIPTION
**Pull request by bruce3557**
(1) Put log in file
(2) Change logging level, please change the logging level manually if you need debug message.

### CODE REVIEW

File: utils/log_info.py [hunk 1]
```diff
@@ -35,18 +36,30 @@ def get_logger(name='NTHU OJ'):
...
+
         # create logger
         logger = logging.getLogger(name)
-        logger.setLevel(logging.DEBUG)
+        logger.setLevel(logging.INFO)
+
+        # create formatter for console use
+        formatter = logging.Formatter(logging_format)
 
         # create console handler and set level to debug
```
Comment (henryyang42) [inline utils/log_info.py]: comment is not consistent with the code XD

**Comment by nickelcheng** [comment]
I don't know I added commits OAQ

**Comment by nickelcheng** [comment]
test ok, +1

**Comment by bruce3557** [comment]
So I want a developer machine QQ
Thanks !

**Comment by bruce3557** [comment]
Fixed, thanks!

**Comment by nickelcheng** [comment]
merge

**Pull request merged**